app/components/scbd-angularjs-services/services/storage.js

```diff
@@ -406,7 +406,20 @@
 
             "getMimeType": function(file) {
                 return getMimeTypes(file.name, file.type || "application/octet-stream");
-            }
+            },
+
+            mimeTypeWhitelist : [
```

Comment:
One day may be we should ahve a central repository for standard values that we use. Mimetype is a good example